gunicorn/config.py

```diff
@@ -368,6 +368,15 @@ def validate_string(val):
     return val.strip()
 
 
+def validate_file_exists(val):
+    val = validate_string(val)
+    if val is None:
```

Comment:
I'm not sure we want this here. For the string options, I think it's there for when the default is `None` if the option wasn't provided at all. In the case of this option, the default is an empty list.